File: 시나공/004 연산자/문제1.c
#include <stdio.h>
int main() {
    int result, a= 100, b=200, c=300;
    result = a < b ? b++ : --c;
    printf("%d, %d, %d\n", result, b, c);


    // 100 < 200 ? 200 : 299
    // 참 ? 200 : 299
    // result = 200, b = 201, c = 300(결과적으로 거짓이 아니였기 때문에 --c이 적용되지 않음)

    // 결과
    // 200, 201, 300
}

File: 시나공/010 Java의 클래스와 메소드/문제1.java
public class Test {
	static int res10() {
		return 4;
	}
	static int res30() {
		return 30 + res10();
	}
	static int res200() {
		return 200 + res30();
	}
	public static void main(String args[]) { //메인
		int result;
		result = res200();
		System.out.print(result);
	}
}

// result = res200() =  200+res30() = 200+30+res10() = 200+30+4 = 234

// 결과 234